---
layout: default
title: Pipes Lab / research
categories:
 - research
---

    <h1>Methods for Environmental Analyses</h1>
    <hr style="height:3px;border:none;color:#808088;background-color:#808088;" />
    <img src="/images/striking_image.png" style="float:right; width: 55vw; min-width: 330px;">

    A core line of research in the group is the development of computational methods for the accurate assignment of environmental DNA (eDNA) samples.  
For example, we have previously developed phylogenetic-based methods for clustering diverse sequences implemented in the 
<a href="https://github.com/lpipes/AncestralClust">AncestralClust</a>. We have also developed <a href="https://github.com/lpipes/tronko">tronko</a>
which is a fast approximate likelihood method for phylogenetic assignment of DNA sequences. tronko is the core algorithm behind <a href="https://www.ednaexplorer.org/">
 eDNA Explorer</a>, which is a platform for sharing, exploring, and analyzing data from projects that use eDNA.

     <div class="bigspacer"></div>

     <h4>Example Papers</h4>
        <ol type = "1">
          <li><a href="/pdfs/papers/elife-85794-v2.pdf">A rapid phylogeny-based method for accurate community profiling of large-scale metabarcoding datasets</a></li>
          <li><a href="/pdfs/papers/btab723.pdf">AncestralClust: clustering of divergent nucleotide sequences by ancestral sequence reconstruction using phylogenetic trees</a></li>
          <li><a href="/pdfs/papers/Methods Ecol Evol - 2019 - Curd - Anacapa Toolkit An environmental DNA toolkit for processing multilocus metabarcode.pdf">Anacapa Toolkit: An environmental DNA toolkit for processing
multilocus metabarcode datasets</a></li>
         </ol>
    <div class="bigspacer"></div>


    <h1>Methods for Wastewater Pathogen Surveillance</h1>
    <hr style="height:3px;border:none;color:#808088;background-color:#808088;" />
    <img src="/images/wastewater_picture.jpeg" style="float:right; width: 35vw; min-width: 330px;">

As the global imperative to continually sequence wastewater becomes more crucial in preemptively tracking the spread of pathogens,
our methods not only solidify our frontline defense against disease but are vital against the undetected spread of new variants 
and enhance our nuanced understanding of disease dynamics. We have focused on tracking SARS-CoV-2 variants previously but our methods 
are broadly applicable to tracking other pathogens. We are currently working on methods for tracking avian flu variants from wastewater.

     <div class="bigspacer"></div>
     <h4>Example Papers</h4>

     <ol type = "1">
      <li><a href="/pdfs/papers/PIIS266723752200203X.pdf">Estimating the relative proportions of SARS-CoV-2 haplotypes from wastewater samples</a></li>
      <li><a href="/pdfs/papers/mgen001249.pdf">Tracking SARS-CoV-2 variants of concern in wastewater: an assessment of nine computational tools using simulated genomic data</a></li>
      </ol>
 <div class="bigspacer"></div>
 <div class="bigspacer"></div>
 <div class="bigspacer"></div>
 <div class="bigspacer"></div>
 <div class="bigspacer"></div>
 <div class="bigspacer"></div>
<div class="bigspacer"></div>
<div class="bigspacer"></div>
<div class="bigspacer"></div>
<div class="bigspacer"></div>

             <h1>Phylogenetic Methods and Evolutionary Inference</h1>
             <hr style="height:3px;border:none;color:#808088;background-color:#808088;" />
             <img src="/images/Picture7.png" style="float:right">
Another key focus in our lab is developing and applying methods for phylogenetic analysis, 
with a particular emphasis on using phylogenetic trees to explore molecular evolution. 
Currently, we are investigating how large-scale phylogenies, with more than 100,000 leaf nodes, 
can provide deeper insights into evolutionary processes.
                 <div class="bigspacer"></div>
                 <h4>Example Papers</h4>

                 <ol type = "1">
                  <li><a href="/pdfs/papers/uncertainty.pdf">Assessing uncertainty in the rooting of the SARS-CoV-2 phylogeny</a></li>
                  </ol>
             <div class="bigspacer"></div>


           
