---
layout: default
title: Pipes Lab / research
categories:
 - research
---

    <h1>Methods for Environmental Analyses</h1>
    <hr style="height:3px;border:none;color:#808088;background-color:#808088;" />
    <img src="/images/striking_image.png" style="float:right; width: 55vw; min-width: 330px;">

    A core line of research in the group is the development of computational methods for the accurate assignment of environmental DNA (eDNA) samples.  
For example, we have previously developed phylogenetic-based methods for clustering diverse sequences implemented in the 
<a href="https://github.com/lpipes/AncestralClust">AncestralClust</a>. We have also developed <a href="https://github.com/lpipes/tronko">tronko</a>
which is a fast approximate likelihood method for phylogenetic assignment of DNA sequences. tronko is the core algorithm behind <a href="https://www.ednaexplorer.org/">
 eDNA Explorer</a>, which is a platform for sharing, exploring, and analyzing data from projects that use environmental DNA.

     <div class="bigspacer"></div>

     <h4>Example Papers</h4>
        <ol type = "1">
          <li><a href="/pdfs/papers/elife-85794-v1.pdf">A rapid phylogeny-based method for accurate community profiling of large-scale metabarcoding datasets</a></li>
          <li><a href="/pdfs/papers/btab723.pdf">AncestralClust: clustering of divergent nucleotide sequences by ancestral sequence reconstruction using phylogenetic trees</a></li>
          <li><a href="/pdfs/papers/Methods Ecol Evol - 2019 - Curd - Anacapa Toolkit An environmental DNA toolkit for processing multilocus metabarcode.pdf">Anacapa Toolkit: An environmental DNA toolkit for processing
multilocus metabarcode datasets</a></li>
         </ol>
    <div class="bigspacer"></div>


    <h1>Methods for Wastewater Pathogen Surveillance</h1>
    <hr style="height:3px;border:none;color:#808088;background-color:#808088;" />
    <img src="/images/wastewater_picture.jpeg" style="float:right; width: 55vw; min-width: 330px;">

As the global imperative to continually sequence wastewater becomes more crucial in preemptively tracking the spread of pathogens,
our methods not only solidify our frontline defense against disease but are vital against the undetected spread of new variants 
and enhance our nuanced understanding of disease dynamics. We have focused on tracking SARS-CoV-2 variants previously but our methods 
are broadly applicable to tracking other pathogens. We are currently working on methods for tracking avian flu variants from wastewater.

     <div class="bigspacer"></div>
     <h4>Example Papers</h4>

     <ol type = "1">
      <li><a href="/pdfs/papers/PIIS266723752200203X.pdf">Estimating the relative proportions of SARS-CoV-2 haplotypes from wastewater samples</a></li>
      <li><a href="/pdfs/papers/mgen001249.pdf">Tracking SARS-CoV-2 variants of concern in wastewater: an assessment of nine computational tools using simulated genomic data</a></li>
      </ol>
 <div class="bigspacer"></div>
 <div class="bigspacer"></div>
 <div class="bigspacer"></div>
 <div class="bigspacer"></div>
 <div class="bigspacer"></div>
 <div class="bigspacer"></div>

             <h1>Phylogenetic inferences from DNA sequences</h1>
             <hr style="height:3px;border:none;color:#808088;background-color:#808088;" />
             <img src="/images/Picture7.png" style="float:right">

             We have worked on methods for phylogenetic analyses, particularly for using
              phylogentic trees for inferences in molecular evolution.  Many of the methods we have helped
              develop are implemented in
              the <a href="http://abacus.gene.ucl.ac.uk/software/#phylogenetic-analysis-by-maximum-likelihood-paml">PAML</a> package
                by <a href="http://abacus.gene.ucl.ac.uk">Ziheng Yang</a>.  We have also worked on modeling expression
                 level evolution along the lineages of a phylogeny and on various statistical methods for
                 imputation using phylogenies.  We are currently working on using large phylogenies
                 (>100k leaf nodes) for inference of evolutionary processes.
                 <div class="bigspacer"></div>
                 <h4>Example Papers</h4>

                 <ol type = "1">
                  <li><a href="/pdfs/papers/uncertainty.pdf">Assessing uncertainty in the rooting of the SARS-CoV-2 phylogeny</a></li>
                    <li><a href="/pdfs/papers/lizard.pdf">Phenotypic convergence is not mirrored at the protein level in a lizard adaptive radiation</a></li>
                      <li><a href="/pdfs/papers/anova.pdf">Phylogenetic ANOVA: The Expression Variance and Evolution Model for Quantitative Trait Evolution</a></li>
                  </ol>
             <div class="bigspacer"></div>
             <div class="bigspacer"></div>


             <h1>Methods for Environmental DNA Analyses</h1>
             <hr style="height:3px;border:none;color:#808088;background-color:#808088;" />
             <img src="/images/Picture8.jpg" style="float:right">

             We have on and off been working on DNA barcoding and analyses of environmental DNA over the past 15 years.
              This is an area of very active research in the lab at the moment.  We focus on developing better methods
               for species assignment and on developing methods for leveraging eDNA for population genetic analyses.
               <div class="bigspacer"></div>
               <h4>Example Papers</h4>

               <ol type = "1">
                <li><a href="/pdfs/papers/barcode.pdf">Fast phylogenetic DNA barcoding</a></li>
                  <li><a href="/pdfs/papers/assign.pdf">Statistical Assignment of DNA Sequences Using Bayesian Phylogenetics</a></li>
                    <li><a href="/pdfs/papers/greenland.pdf">Ancient Biomolecules from Deep Ice Cores Reveal a Forested Southern Greenland</a></li>
                      <li><a href="/pdfs/papers/sediment.pdf">Ancient DNA Chronology within Sediment Deposits: Are Paleobiological Reconstructions Possible and is DNA Leaching a Factor?</a></li>
                </ol>
           <div class="bigspacer"></div>
           <div class="bigspacer"></div>
           <div class="bigspacer"></div>
           <div class="bigspacer"></div>
           <div class="bigspacer"></div>






                <h1>Evolutionary Genomics of Domesticated Plants and Animals</h1>
                <hr style="height:3px;border:none;color:#808088;background-color:#808088;" />
                <img src="/images/Picture9.png" style="float:right">

                We have worked on multiple research aimed at understanding evolutionary processes during domestication,
                particularly the way domestication selection has affected genetic variation and the consequences
                of admixture with wild species.  We are currently working particularly on rice and on betta fish.
                <div class="bigspacer"></div>
                <h4>Example Papers</h4>

                <ol type = "1">
                  <li><a href="/pdfs/papers/highalt.pdf">Ancient hybridization with an unknown population facilitated high-altitude adaptation of canids</a></li>
                    <li><a href="/pdfs/papers/bos.pdf">Pervasive introgression facilitated domestication and adaptation in the Bos species complex</a></li>
                      <li><a href="/pdfs/papers/wildrice.pdf">Asian wild rice is a hybrid swarm with extensive gene flow and feralization from domesticated rice</a></li>
                        <li><a href="/pdfs/papers/power.pdf">The power of inbreeding: NGS based GWAS of rice reveals convergent evolution during rice domestication</a></li>
                 </ol>
            <div class="bigspacer"></div>











                   <h1>Cancer Evolution and Cell Lineage Trees</h1>
                   <hr style="height:3px;border:none;color:#808088;background-color:#808088;" />
                   <img src="/images/Picture10.jpg" style="float:right">

                   Much of our previous work has focused on developing computational methods to elucidate
                    evolutionary changes in DNA or in phenotypes such as expression levels, along the branches
                     of evolutionary trees. We are now using similar computational techniques to elucidate
                     processes along the branches of cell lineage trees within an individual.  The process of
                      differentiation of cells during ontogeny/morphogenesis can be described by a tree, much
                      like an evolutionary tree.  Similarly, the process of mutation and cell differentiation
                       driving carcinogenesis can be described as an evolutionary process acting within an individual
                       in which the unit of evolution is a cell instead of an individual.  A new line of research in
                       the lab is the modeling and analysis of somatic mutations.
                       We have worked with <a href="http://www.bx.psu.edu/makova_lab/">Kateryna Markova</a> on
                       analyzing mitochondrial heteroplasmies and on developing probabilistic models of the spread of
                        somatic mutations during ontogenesis.  These models allow us to take advantage of large
                        NGS data sets to test hypotheses regarding differential proliferation of mutations in different
                         tissues and on natural selection acting on the population of cells within an individual.
                          We are also working on modeling single-cell sequencing data from tumors to provide better
                          computational methods for analyzing tumor evolution.
                          <div class="bigspacer"></div>
                          <h4>Example Papers</h4>

                          <ol type = "1">
                           <li><a href="/pdfs/papers/hair.pdf">Pronounced somatic bottleneck in mitochondrial DNA of human hair</a></li>
                            <li><a href="/pdfs/papers/bottle.pdf">Bottleneck and selection in the germline and maternal age influence transmission of mitochondrial DNA in human pedigrees</a></li>
                              <li><a href="/pdfs/papers/hetero.pdf">A population phylogenetic view of mitochondrial heteroplasmy</a></li>
                           </ol>
                      <div class="bigspacer"></div>
